Code/Framework/AzCore/AzCore/Module/DynamicModuleHandle.h

```diff
@@ -87,8 +107,14 @@ namespace AZ
             AlreadyLoaded
         };
 
+        template<typename T>
+        inline bool CheckBitsAny(T v, T bits)
+        {
+            return (v & bits) != (T)0;
+        }
```

Comment:
This can be marked constexpr
```suggestion
        template<typename T>
        constexpr bool CheckBitsAny(T v, T bits)
        {
            return (v & bits) != T{};
        }
```